backend/x86_gas.ml

```diff
@@ -455,11 +461,13 @@ let print_line b = function
       | _ -> bprintf b ",%s" (String.concat "," args)
       end
   | Space n ->
-      if system = S_solaris then bprintf b "\t.zero\t%d" n
-      else bprintf b "\t.space\t%d" n
+      (match system with
+      | S_solaris -> bprintf b "\t.zero\t%d" n
```

Comment:
do we still need `S_solaris` ?

Reply:
I would rather leave this kind of
clean-up to another pull request.

Reply:
(I have added a CR-sson to `x86_proc.mli`.)